mne/viz/topomap.py

```diff
@@ -1037,7 +1037,7 @@ def plot_evoked_topomap(evoked, times=None, ch_type='mag', layout=None,
     if colorbar:
         cax = plt.subplot(1, n + 1, n + 1)
         plt.colorbar(images[-1], ax=cax, cax=cax, ticks=[vmin, 0, vmax],
-                     format=format)
+                     format=fmt)
```

Comment:
@dengemann  matplotlib and scipy.sparse seem to be using format at different places.
We're not that odd using it in mne so I wonder...